shell.nix

```diff
@@ -34,6 +34,7 @@ let
       pkg-config
       hlint
       ormolu.ormolu
+      haskell-language-server
```

Comment:
This isn't needed! `haskell-language-server` is specified in the tools section below, and is already available!

Reply:
I have no idea what I did with my emacs :smile: , but it works now just fine. I must have fixed my config and added this line at the same time. You are right this line is not necessary, will remove it.